tv-show/src/components/features/shows/ShowContainer/ShowContainer.tsx

```diff
@@ -2,7 +2,7 @@
 import { Container } from "@chakra-ui/react";
 import ShowReviewSection from "../ShowReviewSection/ShowReviewSection";
 import { IShow } from "@/typings/Show.type";
-import { useEffect, useState } from "react";
+import { useEffect } from "react";
```

Comment:
In line 65, you are calling this effect conditionally, which is a no-go for any hook. It has to be defined above the early returns.

Reply:
Looking at it, I am also not sure what it is doing.